src/main/webapp/resources/js/pages/admin/components/AdminUsersPage.jsx

```diff
@@ -13,23 +13,25 @@ import { PagedTableProvider } from "../../../components/ant.design/PagedTable";
 import { setBaseUrl } from "../../../utilities/url-utilities";
 import { AddNewButton } from "../../../components/Buttons/AddNewButton";
 import { UsersTable } from "../../../components/UsersTable/UsersTable";
+import CreateNewUser from "./user/CreateNewUser";
 
 export default function AdminUsersPage() {
   // The following renders the Users component view
   return (
-    <PageWrapper
-      title={i18n("AdminPanel.users")}
-      headerExtras={
-        <AddNewButton
-          className={"t-add-user-btn"}
-          href={setBaseUrl(`users/create`)}
-          text={i18n("AdminPanel.addUser")}
-        />
-      }
-    >
-      <PagedTableProvider url={setBaseUrl("ajax/users/list")}>
+    <PagedTableProvider url={setBaseUrl("ajax/users/list")}>
+      <PageWrapper
+        title={i18n("AdminPanel.users")}
+        headerExtras={[
+          <CreateNewUser />,
+          <AddNewButton
```

Comment:
Don't forget to remove the old files, they can probably be removed now

Reply:
Please see 15cc6830fd0ac7e6aa4199b7b9450b04bd0e83c9.